<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Title</title>
</head>
<body>
<script>
  let p1 = Promise.resolve("hello promise!");
  p1.then(data=>{
    console.log('data', data);
  }).catch(err=>{
    //除非 then 中抛出异常，就会进入到 catch 中，否则不会进入到 catch 中
  })
</script>
</body>
</html>
